.github/workflows/metal-run-microbenchmarks.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
           ./scripts/build_scripts/build_with_profiler_opt.sh
           ./create_venv.sh
       - name: Build tests
-        run: cmake --build build --target tests -- -j`nproc`
```

Comment:
Does this build the CPP tests needed for this pipeline?

Reply:
No it doesn't! Good catch!
The issue is that this workflow leverages a script `build_with_profiler_opt.sh`.
This is odd, because every other workflow directly invokes CMake.
The script runs the CMake configure step, and I just updated it to provide the necessary CMake options.
I am manually triggering the workflow to make sure that we are good on this.

Reply:
This is fixed.